File: customer-java-resttemplate/src/main/java/com/prez/Application.java
package com.prez;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;

@SpringBootApplication
@EnableMongoRepositories(basePackages = {"com.prez.db", "com.prez.model"})
public class Application {

  public static void main(String[] args) {
    SpringApplication.run(Application.class, args);
  }

}

File: customer-kotlin-webflux/src/main/kotlin/com/prez/Application.kt
package com.prez

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker
import org.springframework.web.reactive.config.EnableWebFlux

@SpringBootApplication
@EnableWebFlux
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}